src/main/java/io/gingersnapproject/cdc/cache/hotrod/HotRodService.java

```diff
@@ -91,10 +130,10 @@ public OffsetBackend offsetBackendForURI(URI uri) {
       return new HotRodOffsetBackend(remoteCacheManager.getCache(OFFSET_CACHE_NAME));
    }
 
-   private void getOrCreateOffsetBackendCache(RemoteCacheManager rcm) {
+   private void getOrCreateOffsetBackendCache(String name, RemoteCacheManager rcm) {
       rcm.administration()
             .withFlags(CacheContainerAdmin.AdminFlag.VOLATILE)
-            .getOrCreateCache(OFFSET_CACHE_NAME, new StringConfiguration(
+            .getOrCreateCache(name, new StringConfiguration(
```

Comment:
One thing we need to keep in mind that I hadn't thought about is we need to remove an offset when a rule is no longer configured.

Reply:
I created https://github.com/gingersnap-project/db-syncer/issues/43 encapsulate all the things we need to remove when a rule is removed